src/main/java/org/folio/circulation/domain/configuration/CheckoutLockConfiguration.java

```diff
@@ -0,0 +1,39 @@
+package org.folio.circulation.domain.configuration;
+
+import io.vertx.core.json.JsonObject;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.ToString;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.lang.invoke.MethodHandles;
+
+import static org.folio.circulation.support.json.JsonPropertyFetcher.getBooleanProperty;
+import static org.folio.circulation.support.json.JsonPropertyFetcher.getIntegerProperty;
+
+@AllArgsConstructor
+@Getter
+@ToString
+public class CheckoutLockConfiguration {
+
+  protected static final Logger log = LogManager.getLogger(MethodHandles.lookup().lookupClass());
+  private final boolean isCheckOutLockFeatureEnabled;
+  private final int noOfRetryAttempts;
+  private final int retryInterval;
+  private final int lockTtl;
+
+  public static CheckoutLockConfiguration from(JsonObject jsonObject) {
+    try{
```

Comment:
Please see my other comment about whitespace. Please go through the whole PR and make sure you're following code formatting best practices.